ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/activityServiceAgent.service.ts

```diff
@@ -31,76 +31,119 @@ export class ActivityServiceAgent extends UserServiceAgent<any> {
         private converter: ActivityConverter,
         http: HttpClient,
         logger: LoggingService,
-        private __language: LanguageService) { // eslint-disable-line @typescript-eslint/naming-convention, no-underscore-dangle, id-blacklist, id-match
+        private __language: LanguageService
+    ) {
+        // eslint-disable-line @typescript-eslint/naming-convention, no-underscore-dangle, id-blacklist, id-match
         super(session, configuration, http, logger, null);
     }
 
-    public getActivity(studyGuid$: Observable<string | null>,
-                       activityGuid$: Observable<string | null>): Observable<ActivityForm> {
-
-        const studyGuidEmitted$: Observable<string | null> = this.__language.getProfileLanguageUpdateNotifier()
-            .pipe(
-                switchMap(() => studyGuid$)
-            );
+    public getActivity(
+        studyGuid$: Observable<string | null>,
+        activityGuid$: Observable<string | null>
+    ): Observable<ActivityForm> {
+        const studyGuidEmitted$: Observable<string | null> = this.__language
+            .getProfileLanguageUpdateNotifier()
+            .pipe(switchMap(() => studyGuid$));
 
```

Comment:
Same issue here for the formatting. Still trying to figure out what your REAL code changes are doing and it looks like this is just formatting.